runtime/imported_values_memory_metering_test.go

```diff
@@ -372,7 +372,7 @@ func TestImportedValueMemoryMetering(t *testing.T) {
 
 		executeScript(script, meter, structValue)
 		assert.Equal(t, uint64(1), meter[common.MemoryKindCompositeValueBase])
-		assert.Equal(t, uint64(104), meter[common.MemoryKindRawString])
+		assert.Equal(t, uint64(17), meter[common.MemoryKindRawString])
```

Comment:
😍

Reply:
doctors hate this 1 weird trick!! one RawString lost 87 bytes in just 1 PR!!!!